src/containers/Footer/index.tsx

```diff
@@ -12,29 +13,29 @@ export const Footer = () => {
   );
 };
 
-const FooterContainer = styled.div`
-  display: flex;
-  height: 8rem;
-  padding: 0 8rem;
-  align-items: center;
-  justify-content: space-between;
-  background-color: ${({ theme }) => theme.headerBackground};
-  border-top: ${({ theme }) => theme.border};
-  width: 100%;
-  max-width: 100vw;
-`;
+const FooterContainer = styled('footer')(() => {
+  const { currentTheme } = useCustomTheme();
+  return {
+    display: 'flex',
+    height: '8rem',
+    padding: '0 8rem',
+    alignItems: 'center',
+    justifyContent: 'space-between',
+    backgroundColor: currentTheme.backgroundSecondary,
+    borderTop: currentTheme.border,
+    width: '100vw',
```

Comment:
recommend % units instead of viewport units on child elements that are not body or html tho.

Reply:
https://github.com/defi-wonderland/web3-react-boilerplate/pull/8/commits/a4a80bcb3d05cb213b045a2f40932f8ae160615c